python/external_config/external_command.py

```diff
@@ -221,13 +271,55 @@ def pipeline_configuration_name(self):
         """
         return self._pipeline_config_name
 
+    @property
+    def system_name(self):
+        """
+        The system name for the command
+        """
+        return self._callback_name
+
     @property
     def display_name(self):
         """
         Display name, suitable for display in a menu.
         """
         return self._display_name
 
+    @property
+    def group(self):
+        """
+        Group command belongs to or None if not defined.
+        """
+        return self._group
+
+    @property
+    def is_group_default(self):
+        """
+        True if this command is a default action for a group.
```

Comment:
Not sure I know what a default action for a group is, could you add a tiny explanation?

Reply:
sorry, absolutely. will add.